packages/ui-extensions/src/surfaces/customer-account/api/standard-api/standard-api.ts

```diff
@@ -236,29 +240,26 @@ export interface NavigationCurrentEntryChangeEvent {
   /**
    * Returns the type of the navigation that resulted in the change.
    */
-  navigationType: NavigationType;
+  navigationType?: NavigationTypeString;
   /**
    * Returns the NavigationHistoryEntry that was navigated from.
    */
   from: NavigationHistoryEntry;
 }
 
-export interface StandardExtensionNavigation {
+export interface Navigation {
   /**
    * The navigate() method navigates to a specific URL, updating any provided state in the history entries list.
    */
   navigate: NavigateFunction;
```

Comment:
We still return => void in NavigateFuncton for customer accounts. We havent implemented commited or intercept yet, this should come later